foundations/src/telemetry/tokio/mod.rs

```diff
@@ -0,0 +1,364 @@
+//! Tokio Runtime Metrics
+//!
+//! Foundations provides a helper API for monitoring Tokio runtimes using the
+//! [`foundations metrics api`](crate::telemetry::metrics).
+//! This helper API allows users to track one or more runtimes using a [`RuntimeMonitor`] object and collect their
+//! metrics periodically using [`RuntimeMonitor::record_sample`].
+//!
+//! ## Metrics
+//! | Metric                                           | Source                                                              | Labels (? indicates optional)          |
+//! |:-------------------------------------------------|:--------------------------------------------------------------------|:---------------------------------------|
+//! | tokio_runtime_workers                            | [`tokio::runtime::RuntimeMetrics::num_workers`]                     | runtime_name?, runtime_id?             |
```

Comment:
Let's just make metrics structs public. Yes, users could modify them manually, but it's not a big deal. It will be easier to have regular rust docs for them are in sync with metric info than maintaining with markdown table

Reply:
I really, really, really do not want to make them public as people will absolutely try to edit them manually.

Also, this markdown table is actually quite easy to maintain.